internal/utils/edgedb/types.go

```diff
@@ -23,6 +23,7 @@ type Policy struct {
 
 type PolicyConfig struct {
 	ID          edgedb.UUID        `edgedb:"id"`
+	Name        string             `edgedb:"name"`
 	PolicyName  []Policy           `edgedb:"policyName"`
```

Comment:
If it's a link, the field name should probably just be `policy` now. But if you have something in mind for this pending the "policy doesn't exist at config creation time" thing then fine to keep it this way for now. But it probably isn't an array, in any case (?)

Reply:
I only kept it in case multiple Configs would point to the same Policy, but it depends on how the final implementation of Configs would be. 

The CR itself will always have a name and a PolicyName so I just translated that to here.